src/idvc/dvc_interface.py

```diff
@@ -428,8 +428,15 @@ def CreateHelpPanel(self):
             "9. Once satisfied with the registration, move to the next tab. Be aware that point 0 is the point from which DVC will start."
             )
         
-        self.help_text.append("To create a mask you need to create a selection. Start tracing a freehand region for the selection by clicking 'Start Tracing' button.\n"
-            "When you are happy with your region click 'Create Mask'.")
+        self.help_text.append(
+            "1. If you already have a mask file, upload it.\n"
+            "2. To create a new mask click 'Start tracing'. Click on the image and trace a freehand region while holding the mouse button.\n"
```

Comment:
How to use the [tracing widget](https://vtk.org/doc/nightly/html/classvtkImageTracerWidget.html#details):

The button actions and key modifiers are as follows for controlling the widget: 

1) left button click over the image, hold and drag draws a free hand line. 
2) left button click and release erases the widget line, if it exists, and repositions the first handle. 
3) middle button click starts a snap drawn line. The line is terminated by clicking the middle button while depressing the ctrl key. 
4) when tracing a continuous or snap drawn line, if the last cursor position is within a specified tolerance to the first handle, the widget line will form a closed loop. 
5) right button clicking and holding on any handle that is part of a snap drawn line allows handle dragging: existing line segments are updated accordingly. The path can be closed by repositioning the first and last points over one another. 
6) ctrl key + right button down on any handle will erase it: existing snap drawn line segments are updated accordingly. If the line was formed by continuous tracing, the line is deleted leaving one handle. 
7) shift key + right button down on any snap drawn line segment will insert a handle at the cursor position. The line segment is split accordingly.

Reply:
I think we should add these instructions, thanks. I need to think where the best place to insert them would be

Reply:
We discussed this with Brian on 06/06/24 and he also agreed it was a great feature to emphasize.